panel/panel.py

```diff
@@ -163,30 +162,18 @@ def filter_ok(check_val, key, panel_filter):
 
 
 def get_view_lines(view_dict, panel_filter):
-    view_lines = []
-    for lineno, line_dict in sorted(view_dict["line_dicts"].items()):
-        items = []
-        for error_type in WARN_ERR:
-            if not filter_ok(error_type, "types", panel_filter):
-                continue
+    return sorted(
+        (item
+         for lineno, line_dict in view_dict["line_dicts"].items()
+         for error_type, items in line_dict.items()
+         for item in items
```

Comment:
This block is nice bc if you change the receiving data structure, you just have to mangle the for..in unpacking here. The rest stays as is.

E.g. if you would consume `raw_errors`, you just delete the first two for..ins and basically just do a `for item in raw_errors[buffer_id]`. And you're done.

Reply:
I'm done for today. 🥃

Reply:
Cheers!